src/Bonsai.ML.Visualizers/TimeSeriesOxyPlotBase.cs

```diff
@@ -41,6 +43,16 @@ internal class TimeSeriesOxyPlotBase : UserControl
         /// </summary>
         public int Capacity { get; set; }
 
+        public ToolStripComboBox ComboBox
+        {
+            get => comboBox;
+        }
+
+        public StatusStrip StatusStrip
+        {
+            get => statusStrip;
+        }
+
```

Comment:
```suggestion
        public ToolStripComboBox ComboBox => comboBox;

        public StatusStrip StatusStrip => statusStrip;
```

For getter-only properties it is possible to go even simpler.

Reply:
Updated